recipes/fandango/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "fandango" %}
+{% set version = "16.0.0" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/fandango-{{ version }}.tar.gz
+  sha256: a4530b04d67f8b42e6f4924ded1ba146a63c4d1fd2fcae2584d36c64bf94a625
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools    
+    - future    
+  run:
+    - python >={{ python_min }}
+    - future
+
+test:
+  imports:
+    - fandango
+  commands:
+    - pip check
```

Would be nice to test some entry points here. I'd add:

```suggestion
    - pip check
    - fandango --help
```

I guess other entry points require extra dependencies, so you can skip them.